README.adoc

```diff
@@ -92,7 +92,7 @@ To install g2p-seq2seq :
 
     python setup.py install
     
-You will also need to install http://www.perl.org/get.html[Perl] and move install/quick_lm.pl to the same directory as the CCAligner or a dictionary that is set in the environment variable PATH
+You will also need to install http://www.perl.org/get.html[Perl] and move install/quick_lm.pl to the same directory as the CCAligner or a dictionary that is set in the environment variable (PATH)
```

Comment:
Though the change you sent is not actually required, there's another mistake in this line 😛  Do you want me to point it out, or you want to find it yourself 😉